app/ide-desktop/lib/dashboard/e2e/actions.ts

```diff
@@ -495,75 +510,89 @@ export function locateCollapsibleDirectories(page: test.Page) {
 }
 
 /** Find a "confirm delete" modal (if any) on the current page. */
-export function locateConfirmDeleteModal(page: test.Locator | test.Page) {
+export function locateConfirmDeleteModal(page: test.Page) {
   // This has no identifying features.
   return page.getByTestId('confirm-delete-modal')
 }
 
 /** Find a "new label" modal (if any) on the current page. */
-export function locateNewLabelModal(page: test.Locator | test.Page) {
+export function locateNewLabelModal(page: test.Page) {
   // This has no identifying features.
   return page.getByTestId('new-label-modal')
 }
 
 /** Find an "upsert secret" modal (if any) on the current page. */
-export function locateUpsertSecretModal(page: test.Locator | test.Page) {
+export function locateUpsertSecretModal(page: test.Page) {
   // This has no identifying features.
   return page.getByTestId('upsert-secret-modal')
 }
 
 /** Find a user menu (if any) on the current page. */
-export function locateUserMenu(page: test.Locator | test.Page) {
+export function locateUserMenu(page: test.Page) {
   // This has no identifying features.
   return page.getByTestId('user-menu')
 }
 
 /** Find a "set username" panel (if any) on the current page. */
-export function locateSetUsernamePanel(page: test.Locator | test.Page) {
+export function locateSetUsernamePanel(page: test.Page) {
   // This has no identifying features.
   return page.getByTestId('set-username-panel')
 }
 
 /** Find a set of context menus (if any) on the current page. */
-export function locateContextMenus(page: test.Locator | test.Page) {
+export function locateContextMenus(page: test.Page) {
   // This has no identifying features.
   return page.getByTestId('context-menus')
 }
 
 /** Find a labels panel (if any) on the current page. */
-export function locateLabelsPanel(page: test.Locator | test.Page) {
+export function locateLabelsPanel(page: test.Page) {
   // This has no identifying features.
   return page.getByTestId('labels')
 }
 
 /** Find a list of labels (if any) on the current page. */
-export function locateLabelsList(page: test.Locator | test.Page) {
+export function locateLabelsList(page: test.Page) {
   // This has no identifying features.
   return page.getByTestId('labels-list')
 }
 
 /** Find an asset panel (if any) on the current page. */
-export function locateAssetPanel(page: test.Locator | test.Page) {
+export function locateAssetPanel(page: test.Page) {
   // This has no identifying features.
   return page.getByTestId('asset-panel')
 }
 
 /** Find a search bar (if any) on the current page. */
-export function locateSearchBar(page: test.Locator | test.Page) {
+export function locateSearchBar(page: test.Page) {
   // This has no identifying features.
   return page.getByTestId('asset-search-bar')
 }
 
+/** Find an extra columns button panel (if any) on the current page. */
+export function locateExtraColumns(page: test.Page) {
+  // This has no identifying features.
```

Comment:
Not sure what this comment means or if it's useful. Seems to appear in every function?

Reply:
every function with `getByTestId`. the intention is that Playwright's philosophy is to perform tests like how a user would see it - alt text, tooltip, role (button/listitem/something else). the comments are probably not useful though... but they're supposed to serve as a reminder that this isn't ideal